tests/Helpers/Models/MethodConfigTest.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace Moves\TestHelpers\Models;
+
+use Moves\Eloquent\Validatable\Interfaces\IValidatable;
+use Moves\Eloquent\Validatable\Validatable;
+
+class MethodConfigTest implements IValidatable
```

Comment:
I don't love to pick on people for naming conventions because everybody has something different that they're comfortable with, but I do have a complaint here. As a commonly shared convention, ending the classname with `Test` implies that this class itself is actually executing test cases, which in this case it's not. This is just a supporting class to make the actual test cases easier to write.

Usually, I like to call my helper models something like `ValidatableModel`, i.e. the name of the trait you're testing + `Model`. In your case, since you'll need two separate copies, I'd call them `ValidatableFunctionsModel` and `ValidatableAttributesModel`.

With that said, I am interested to explore alternate naming conventions, because `Model` feels a bit redundant given that the namespace is already `Tests\Helpers\Models`. Maybe just abbreviate it with an `M` prefix, kind of in the same way that some people (including myself) prefix interfaces with `I`? So like `MValidatableFunctions` and `MValidatableAttributes`? Feel free to explore what you like best - just don't end it with `Test` :)

Reply:
Agreed!